### ISSUE TITLE
local_auth package getAvailableBiometrics in different android versions

### ISSUE THREAD

**Issue by TarekMedhat** [flutter/flutter#73222]
In local_auth package documentation it's mentioned that:
_On Android, you can check only for existence of fingerprint hardware prior to API 29 (Android Q). Therefore, if you would like to support other biometrics types (such as face scanning) and you want to support SDKs lower than Q, do not call getAvailableBiometrics. Simply call authenticateWithBiometrics. This will return an error if there was no hardware available._

What does that exactly mean? this part is totally unclear.
Wasn't Biometric faceId support added to android as part of BiometricPrompt only after Android Q, Why would that be an issue for calling getAvailableBiometrics in lower than Q versions?

**Comment by iapicca** [comment]
> What does that exactly mean? this part is totally unclear.
> Wasn't Biometric faceId support added to android as part of BiometricPrompt only after Android Q, Why would that be an issue for calling getAvailableBiometrics in lower than Q versions?

@TarekMedhat 
given that [faceId](https://en.wikipedia.org/wiki/Face_ID) doesn't anything to do with android
[local_auth](https://pub.dev/packages/local_auth)  docs simply explain what to do if android version is less then `Q`

> if [...] you want to support SDKs lower than Q, do not call `getAvailableBiometrics`. Simply call `authenticateWithBiometrics`

i fail to understand what isn't clear

**Comment by pedromassangocode** [comment]
Hi @TarekMedhat 
Can you elaborate  what this issue is about? It is not clear to me as well.
Thank you

**Comment by Purvik** [comment]
@TarekMedhat  @iapicca  If I follow same as suggested in document, not to use `getAvailableBiometrics()` and directly perform authentication using `authenticateWithBiometrics`, application is still not asking for Face recognition only finger authentication. For an update, permission 'USES_FINGERPRINT' is deprecated in API 28 and instead of it we have to use 'USES_BIOMETRIC' which I tried but got the same result.

**Comment by pedromassangocode** [comment]
Without additional information, we are unfortunately not sure how to resolve this issue. 
We are therefore reluctantly going to close this bug for now. 
Please don't hesitate to comment on the bug if you have any more information for us; we will reopen it right away!  
Thanks for your contribution.

Could everyone who still has this problem please file a new issue with the exact description of what happens, logs, and the output of `flutter doctor -v`.
All system setups can be slightly different, so it's always better to open new issues and reference related issues.
